frontend/src/components/Login/Login.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react"
+import basestyle from "../base.module.css"
+import loginstyle from "./login.module.css"
+import axios from "axios"
+import { useNavigate, NavLink } from "react-router-dom"
+const Login = () => {
+  const navigate = useNavigate()
+  const [formErrors, setFormErrors] = useState({})
+  const [user, setUserDetails] = useState({
+    email: "",
+    password: "",
+  })
+  const [error, setError] = useState("")
+
+  const changeHandler = (e) => {
+    const { name, value } = e.target
+    setUserDetails({
+      ...user,
+      [name]: value,
+    })
+  }
+  const validateForm = (values) => {
+    const error = {}
+    const regex = /^[^\s+@]+@[^\s@]+\.[^\s@]{2,}$/i
```

Comment:
regex?? Can you please change the name and have a sensible name?

Reply:
changed to "matchPattern"

Reply:
changed to emailRegex